src/lib/divider/divider-module.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {NgModule} from '@angular/core';
+import {MatCommonModule} from '@angular/material/core';
+import {
+  MatDivider,
+  MatDividerCssMatStyler,
+} from './divider';
```

Comment:
nit: you can join this import in a single line and also move the `@angular/common` to the top.

Reply:
I moved the `@angular/common` imports, but the multi-line import is based on precedent in other modules (and offers no benefit to change, since it all gets whitespaced away later)